cmd/utils.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Google LLC
```

Comment:
We already have a `common/utils/utils.go` class, can we rename this and place this outside the `cmd` folder. The `cmd` folder is the first-level folder for all the sub-commands HB exposes. Let's preserve that.

Reply:
Putting these methods in utils/utils.go was creating an import cycle